pilot/pkg/model/service.go

```diff
@@ -329,6 +329,12 @@ func (si *ServiceInstance) GetAZ() string {
 	return si.Labels[AZLabel]
 }
 
+// ServiceAttributes represents a group of custom attributes of the service.
+type ServiceAttributes struct {
+	Name      string
+	Namespace string
+}
```

Comment:
at some point, service accounts should also move into this.

Reply:
Service accounts are attributes on workloads, not services...